proof/invariant-abstract/AARCH64/ArchVSpaceEntries_AI.thy

```diff
@@ -31,7 +31,7 @@ lemmas obj_valid_vspace_simps[simp]
         [split_simps Structures_A.kernel_object.split
                      arch_kernel_obj.split]
 
-abbreviation
+locale_abbrev
```

Comment:
Would I be able to get a quick reminder about what `locale_abbrev` is?

Reply:
abbreviations defined in locales usually don't contract (are input-only). That makes sense for a general setting where the abbreviation might not work outside the locale. `locale_abbrev` does contract inside the locale.

Reply:
Thanks!